api/envoy/extensions/filters/http/oauth2/v3/oauth.proto

```diff
@@ -25,6 +25,53 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // [#extension: envoy.filters.http.oauth2]
 //
 
+message CookieConfig {
+  enum CookieType {
+    UNSPECIFIED = 0;
+    BEARER_TOKEN = 1;
+    OAUTH_HMAC = 2;
+    OAUTH_EXPIRES = 3;
+    ID_TOKEN = 4;
+    REFRESH_TOKEN = 5;
+    OAUTH_NONCE = 6;
+  }
+
+  enum SameSite {
+    DISABLED = 0;
+    STRICT = 1;
+    LAX = 2;
+    NONE = 3;
+  }
+
+  // The type of cookie. Must be specified.
+  CookieType type = 1;
+
+  // The value used for the SameSite cookie attribute. If not specified,
+  // defaults to DISABLED which means SameSite value will be missing.
+  SameSite same_site = 2;
+}
+
+// [#next-free-field: 7]
+message CookieConfigs {
+  // Configuration for bearer token cookie
```

Comment:
nit: a period (`.`) at the end of each sentence please, just for consistency, here and below.

Also for here and below, s/for/for the

Reply:
adjusted